packages/suite/src/support/workers/graph/index.ts

```diff
@@ -20,4 +20,4 @@ ctx.addEventListener('message', (event: CustomMessageEvent) => {
 });
 
 // // Trickery to fix TypeScript since this will be done by "worker-loader"
-export default {} as typeof Worker & (new () => Worker);
```

Comment:
This was incorrect IMHO. `webpack-loader` returns the factory not "class". I checked the compiled code. So this is correct typing.